FESS-GUI/mainwindow.cpp

```diff
@@ -557,7 +562,7 @@ void MainWindow::on_actionLock_frame_rate_at_30FPS_triggered(bool checked)
 {
     if(checked)
     {
-        graphRefreshRate = 30;
+        graphRefreshRate = 33;
```

Comment:
Refresh Rates are now saved as FPS/HZ. The original 30 would be correct. 1000/30 is done in a function to get the 33ms for the timer.